src/ru/skypro/Main.java

```diff
@@ -1,8 +1,77 @@
 package ru.skypro;
-
 public class Main {
-
     public static void main(String[] args) {
-	// write your code here
+        int android = 1;
+        int ios = 2;
+        int years = 2014;
+        boolean a = android == 1 && years < 2015;
+        boolean i = ios == 2 && years < 2015;
+        if (i || a) {
+            System.out.println("Установите облегченную версию приложения для ios по ссылке                      " +
+                    "Установите облегченную версию приложения для Android по ссылке");
+        } else {
+            System.out.println("Установите  приложения для Android по ссылке        " + "Установите  приложения для ios по ссылке     ");
+        }
+        {
+            int year = 2000;
+            if(year % 4 == 0 &&  year % 1000 != 0)
+            { System.out.println(year + " високосный");}
+            else
+            { System.out.println(year + " не високосный");}
+        }
+        {
+            int distance = 61;
+            int days = 1;
+            if (distance <= 20){
+                System.out.println("Доставка займет " + days + " сутки");}
+            else if(distance > 20 && distance <=60){
```

Comment:
левые условия автоматически верны из-за невыполнения вышележащих
```suggestion
            else if(distance <=60){
```